Makefile

```diff
@@ -9,6 +9,18 @@ check: depends
 depends:
 	$(foreach project, $(PROJECTS), $(MAKE) -C $(project) $@ || exit 1;)
 
+run:
+	docker run --net host --rm -ti \
```

Comment:
why do you specify the network host?

Reply:
This is because the Azure Functions loader doesn't know (last I checked) won't bind to 0.0.0.0 and instead only binds to localhost